actionpack/lib/action_dispatch/system_testing/driver.rb

```diff
@@ -23,15 +23,19 @@ def use
       end
 
       private
+        def name
+          @name ||= "#{@driver_type}-#{object_id}"
+        end
```

Comment:
Since `ActionDispatch::SystemTesting::Driver` is marked `:nodoc:`, I don't think users are meant to interact with its API directly.

That said, `ActionDispatch::SystemTestCase` doesn't right now offer an ergonomic way to temporarily switch drivers or session outside of the `driven_by` interface. `driven_by` only really works on a per-class basis, so you wanted something more granular, you'd have to use Capybara's API directly. Without a way to get the driver name, I understand that'd be cumbersome. Instead, I think offering the ability for users to specify the Capybara driver name explicitly makes sense 97313e0.